### PR TITLE
zh-translation:/docs/ops/setup/webhook/index.md

### PR DESCRIPTION
**Pull request by TomatoAres**
ref: https://github.com/servicemesher/istio-official-translation/issues/1054

### CODE REVIEW

File: content/zh/docs/ops/setup/webhook/index.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 title: 动态准入 Webhook 概述
-description: 简要描述 Istio 对 Kubernetes webhooks 的使用以及可能出现的相关问题
+description: 简要描述 Istio 对 Kubernetes webhooks 的使用以及可能出现的相关问题。
```
Comment (rootsongjc) [inline content/zh/docs/ops/setup/webhook/index.md]: webhooks -> webhook

**Comment by TomatoAres** [comment]
Close this  pr and restart new pr

**Pull request closed without merging**